### ISSUE TITLE
The date field does not work correctly 

### ISSUE THREAD

**Issue by Juniors017** [decaporg/decap-cms#7166]
I qualify the date field with :

`  - label: "Date"
      name: "Date"
      widget: datetime
      date_format: "DD.MM.YYYY" # e.g. 24.12.2021
      hint: 'Date de publication' 
`
If today date is 03/04/2024 (DD/MM/YYYY)
I select date "now" but decapCMS add 04/03/2024  Day and Month are reverse
![image](https://github.com/decaporg/decap-cms/assets/97809069/9215aee3-8118-434b-935c-5f124e912f8a)
But in preview is OK 
![image](https://github.com/decaporg/decap-cms/assets/97809069/ba8caa28-fb38-4185-a871-128d227351c3)
in frontmatter is KO

i use this script for use decap :  <script src="https://unpkg.com/decap-cms@^3.0.0/dist/decap-cms.js"></script>

**Comment by demshy** [comment]
should be fixed with 3.1.9 version, could you please try again and confirm?

**Comment by Juniors017** [comment]
> should be fixed with 3.1.9 version, could you please try again and confirm?
if you use this it's OK
`- label: "Date de création"
      name: "date"
      widget: datetime
      default: ""
      hint: 'Date de création'  
      format: 'L'`
if you use this it's OK
`- label: "Date de création"
      name: "date"
      widget: datetime
      default: ""
      hint: 'Date de création'  
      format: 'LLL'
                 `
  and  if you use this it's OK 
`- label: "Date"
      name: "Date"
      widget: datetime
      date_format: "DD.MM.YYYY" # e.g. 24.12.2021
      hint: 'Date de publication'`
    
    thanks

**Comment by Juniors017** [comment]
> should be fixed with 3.1.9 version, could you please try again and confirm?

if you use this it's OK

```
- label: "Date de création"
      name: "date"
      widget: datetime
      default: ""
      hint: 'Date de création'  
      format: 'L'
```

if you use this it's OK

```
- label: "Date de création"
      name: "date"
      widget: datetime
      default: ""
      hint: 'Date de création'  
      format: 'LLL'
 ```

  
and  if you use this it's OK 


```
- label: "Date"
      name: "Date"
      widget: datetime
      date_format: "DD.MM.YYYY" # e.g. 24.12.2021
      hint: 'Date de publication'
```
    

thanks

**Comment by adrian5** [comment]
Is this really fixed? It doesn't work for me on DecapCMS 3.1.11, I still get the U.S. date format:

![datetime](https://github.com/user-attachments/assets/06d9e6a0-4ca0-4c11-8ba4-54196b719386)

```yaml
fields:
  - { label: "Title", name: "title", widget: "string"}
  - {
      label: "Begins",
      name: "begins",
      widget: "datetime",
      format: "YYYY-MM-DDTHH:mm:ss",
      date_format: "DD.MM.YYYY", # ignored?
      time_format: "HH:mm" # ignored?
    }
```

And with #7250 it seems the whole datetime logic is still somewhat broken. This used to work fine in the days of NetlifyCMS.

**Comment by martinjagodic** [comment]
@adrian5 see #7250, there is a mismatch between the docs and implementation. I am closing this issue in favor of that one.

**Comment by adrian5** [comment]
@martinjagodic Issue #7250 seems to focus on the storage format (backend) though. It seems to me I cannot configure the UI (frontend) anymore as the documentation claims, and NetlifyCMS was able to.

**Comment by martinjagodic** [comment]
@adrian5 you cannot alter the display format with CMS config. It's the HTML datetime-local input, so the display format is based on the user's browser locale.

**Comment by adrian5** [comment]
@martinjagodic Hmm, I see. I could swear this used to work just fine in NetlifyCMS.

What is the meaning of the documentation?

> `date_format`: sets date display format in UI; boolean or [Day.js formats](https://day.js.org/docs/en/display/format). If used without `time_format`, only the date picker is displayed.

**Comment by martinjagodic** [comment]
You have a point here, thanks for raising it. I will change the docs to clarify that it's the frontmatter format and that the display format is based on the browser locale.

> I could swear this used to work just fine in NetlifyCMS.

I believe you. We had to update it because it used some unmaintained dependencies and affected other areas of the CMS. I opted for native HTML inputs to avoid such problems in the future.
